djangoproject/templates/aggregator/ecosystem.html

```diff
@@ -0,0 +1,160 @@
+{% extends "base_community.html" %}
+{% load i18n %}
+
+{% block content %}
+
+
+  <h1>Django's Ecosystem</h1>
+
+  <p>Django's <em>Secret Sauce</em> is its ecosystem of third-party apps and add-ons.</p>
+  <p>No matter your need, there'll be a package you can use or take ideas from.</p>
+
+  <h2>Resources</h2>
+
+    <!-- Name, Short Description, Link — Roughly this order       -->
+  <ul>
+    <li>
+      <a href="https://awesomedjango.org">Awesome Django</a>
+      &mdash; A curated list of awesome things related to Django.
+    </li>
+    <li>
+      <a href="https://djangopackages.org">Django Packages</a>
+      &mdash; a directory of reusable apps, sites, tools, and more for your Django projects.
+    </li>
+
+    <li>
+      <a href="https://jazzband.co/projects">Jazzband</a>
+      &mdash; a collaborative community to share the responsibility of maintaining Python-based (but mainly Django) projects.
+    </li>
+    <li>
+      <a href="https://github.com/django-commons">Django Commons</a>
+      &mdash; an organization dedicated to supporting the community's efforts to maintain packages.
+    </li>
+    <li>
+      <a href="https://django-news.com">Django News</a>
+      &mdash; Weekly Django news, articles, projects, and more.
+    </li>
+    <li>
+      <a href="https://djangochat.com">DjangoChat Podcast</a>
+      &mdash; The OP podcast on the Django Web Framework
```

Comment:
```suggestion
      &mdash; A podcast on the Django Web Framework and the community.
```

I'm not sure what OP means in this context.